### PR TITLE
CSI Driver v2

### PR DESCRIPTION
**Pull request by dfds-pwes**
Contains all changes required to allow us to move from the early releases of the CSI Driver to version 2.x.x and beyond.

### CODE REVIEW

**Comment by avnes** [comment]
I suppose .ionide/symbolCache.db is not supposed to be commited to Git :)

**Review by avnes — CHANGES_REQUESTED** [review]
Remove binary file symbolCache.db

**Review by avnes — APPROVED** [review]

File: _sub/compute/helm-ebs-csi-driver/main.tf [hunk 1]
```diff
@@ -1,39 +1,179 @@
...
+            "Resource": "*",
+            "Condition": {
+                "StringLike": {
+                    "ec2:ResourceTag/ebs.csi.aws.com/cluster": "true"
+                }
+            }
+        }
+    ]
 }
 EOF
```
[the remark this answers is not in the document]
Comment (dfds-pwes) [reply _sub/compute/helm-ebs-csi-driver/main.tf:158]: Ignore added in the code as this is a vendor provider policy.

[the remark this answers is not in the document]
Comment (dfds-pwes) [reply _sub/compute/helm-ebs-csi-driver/main.tf:158]: Ignore added in the code as this is a vendor provider policy.

**Review by avnes — APPROVED** [review]

**Review by samidbb — APPROVED** [review]

**Review by rifisdfds — APPROVED** [review]

**Pull request merged**